.gitignore

```diff
@@ -5,3 +5,6 @@
 /api/nginx/*.pem
 /bin
 /third_party
+
+# System Files
+.DS_Store
```

Comment:
I would say it's better to have it in the first section and in the same style.

Reply:
Not sure I'm following regarding style.

Reply:
As per our call `.DS_Store` could be located in the subfolders, so leaving as is.